src/main/java/ava/exception/NoCommandException.java

```diff
@@ -0,0 +1,15 @@
+package ava.exception;
+
+/**
+ * Class to represent Duke.Exception.NoCommandException.
```

Comment:
Is this a typo in the documentation? Should this be `Ava.Exception.NoCommandException`?

```suggestion
 * Class to represent Ava.Exception.NoCommandException.
```